packages/loader/driver-utils/src/adapters/compression/compressionTypes.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export enum SummaryCompressionAlgorithm {
+	None = 1,
+	LZ4 = 2,
+}
```

Comment:
@justus-camp - Should we add 'none' and reuse the existing enum for compression algorithms?

/**
 * Available compression algorithms for op compression.
 */
export enum CompressionAlgorithms {
	lz4 = "lz4",
}